### PR TITLE
tests: update tests for floor and ceil intrinsic

### PR DESCRIPTION
**Pull request by parth121101**

### CODE REVIEW

File: integration_tests/intrinsics_262.f90 [hunk 1]
```diff
@@ -0,0 +1,36 @@
+program intrinsics_132
```
Comment (HarshitaKalani) [inline integration_tests/intrinsics_262.f90:1]:
```suggestion
program intrinsics_262
```

File: integration_tests/intrinsics_261.f90 [hunk 2]
```diff
@@ -0,0 +1,36 @@
+program intrinsics_133
```
Comment (HarshitaKalani) [inline integration_tests/intrinsics_261.f90:1]:
```suggestion
program intrinsics_261
```

**Comment by HarshitaKalani** [comment]
I'm getting this failure too.
```
**********************************************************************
** Visual Studio 2019 Developer Command Prompt v16.11.36
** Copyright (c) 2021 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64'
Traceback (most recent call last):
  File "C:\Users\runneradmin\micromamba\envs\lf\Scripts\xonsh-script.py", line 10, in <module>
    sys.exit(main())
  File "C:\Users\runneradmin\micromamba\envs\lf\lib\site-packages\xonsh\main.py", line 504, in main
    _failback_to_other_shells(args, err)
  File "C:\Users\runneradmin\micromamba\envs\lf\lib\site-packages\xonsh\main.py", line 473, in _failback_to_other_shells
    raise err
  File "C:\Users\runneradmin\micromamba\envs\lf\lib\site-packages\xonsh\main.py", line 501, in main
    args = premain(argv)
  File "C:\Users\runneradmin\micromamba\envs\lf\lib\site-packages\xonsh\main.py", line 443, in premain
    start_services(shell_kwargs, args, pre_env=pre_env)
  File "C:\Users\runneradmin\micromamba\envs\lf\lib\site-packages\xonsh\main.py", line 364, in start_services
    XSH.load(ctx=ctx, execer=execer, inherit_env=shell_kwargs.get("inherit_env", True))
  File "C:\Users\runneradmin\micromamba\envs\lf\lib\site-packages\xonsh\built_ins.py", line 616, in load
    from xonsh.commands_cache import CommandsCache
  File "C:\Users\runneradmin\micromamba\envs\lf\lib\site-packages\xonsh\commands_cache.py", line [22](https://github.com/lfortran/lfortran/actions/runs/9577692502/job/26406401526?pr=4295#step:9:23), in <module>
    from case_insensitive_dict import CaseInsensitiveDict as CacheDict
ModuleNotFoundError: No module named 'case_insensitive_dict'
Error: Process completed with exit code 1.
```
I think this is unrelated to this PR.

**Comment by parth121101** [comment]
> I'm getting this failure too.
> ```
> **********************************************************************
> ** Visual Studio 2019 Developer Command Prompt v16.11.36
> ** Copyright (c) 2021 Microsoft Corporation
> **********************************************************************
> [vcvarsall.bat] Environment initialized for: 'x64'
> Traceback (most recent call last):
>   File "C:\Users\runneradmin\micromamba\envs\lf\Scripts\xonsh-script.py", line 10, in <module>
>     sys.exit(main())
>   File "C:\Users\runneradmin\micromamba\envs\lf\lib\site-packages\xonsh\main.py", line 504, in main
>     _failback_to_other_shells(args, err)
>   File "C:\Users\runneradmin\micromamba\envs\lf\lib\site-packages\xonsh\main.py", line 473, in _failback_to_other_shells
>     raise err
>   File "C:\Users\runneradmin\micromamba\envs\lf\lib\site-packages\xonsh\main.py", line 501, in main
>     args = premain(argv)
>   File "C:\Users\runneradmin\micromamba\envs\lf\lib\site-packages\xonsh\main.py", line 443, in premain
>     start_services(shell_kwargs, args, pre_env=pre_env)
>   File "C:\Users\runneradmin\micromamba\envs\lf\lib\site-packages\xonsh\main.py", line 364, in start_services
>     XSH.load(ctx=ctx, execer=execer, inherit_env=shell_kwargs.get("inherit_env", True))
>   File "C:\Users\runneradmin\micromamba\envs\lf\lib\site-packages\xonsh\built_ins.py", line 616, in load
>     from xonsh.commands_cache import CommandsCache
>   File "C:\Users\runneradmin\micromamba\envs\lf\lib\site-packages\xonsh\commands_cache.py", line [22](https://github.com/lfortran/lfortran/actions/runs/9577692502/job/26406401526?pr=4295#step:9:23), in <module>
>     from case_insensitive_dict import CaseInsensitiveDict as CacheDict
> ModuleNotFoundError: No module named 'case_insensitive_dict'
> Error: Process completed with exit code 1.
> ```
> I think this is unrelated to this PR.

Yes @HarshitaKalani , even I think this is not related to my PR. There are other PRs too facing this issue.

**Comment by certik** [comment]
The Windows failure was fixed by https://github.com/lfortran/lfortran/pull/4310.

**Review by certik — APPROVED** [review]

**Pull request merged**